rust/Cargo.toml

```diff
@@ -52,6 +52,8 @@ arrow-ord = "43.0"
 arrow-row = "43.0"
 arrow-schema = "43.0"
 arrow-select = "43.0"
+async-recursion = "1.0"
+async-trait = "0.1.60"
```

Comment:
Did you end up using either of these? I don't see them used.

Reply:
Oh, i did not. It became just general clean up.

Reply:
Oh I see now.